### PR TITLE
New tool: parabricks/dbsnp

### PR DESCRIPTION
**Pull request by Furentsu**
- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -309,7 +309,9 @@ jobs:
             tags: parabricks/haplotypecaller
           - profile: conda
             tags: parabricks/indexgvcf
-          - profile: conda
+          - profile: "conda"
+            tags: parabricks/dbsnp
```
Comment (maxulysse) [inline .github/workflows/test.yml:313]: can we have that properly sorted?

Comment (Furentsu) [reply .github/workflows/test.yml]: Right, I've just fixed it. Still having troubles with pytest though

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -309,7 +311,7 @@ jobs:
             tags: parabricks/haplotypecaller
           - profile: conda
             tags: parabricks/indexgvcf
-          - profile: conda
+          - profile: "conda"
```
Comment (maxulysse) [inline .github/workflows/test.yml:314]:
```suggestion
          - profile: conda
```

**Comment by bsiranosian** [comment]
@maxulysse I think this one is good to go now.

**Review by maxulysse — APPROVED** [review]

**Pull request merged**